test/unit/view/drag-handle.spec.js

```diff
@@ -353,45 +413,84 @@ describe('drag handle', () => {
       });
 
       describe('cancelled before moved enough', () => {
-        describe('cancelled with escape', () => {
-          beforeEach(() => {
-            mouseDown(wrapper, origin, auxiliaryButton);
-            // not moved enough yet
-            windowMouseMove({ x: 0, y: sloppyClickThreshold - 1 });
-            windowEscape();
+        describe('cancelled with any keydown', () => {
+          Object.keys(keyCodes).forEach((key: string) => {
+            describe(`with the ${key} key`, () => {
+              it('should not call execute any callbacks', () => {
```

Comment:
Typo I think, one of call/execute should be removed?

Reply:
fixed